website/docs/tutorials/08-cairo-1-support/03-cheatcodes/README.md

```diff
@@ -0,0 +1,16 @@
+# Cheatcodes
+
+Most of the time, testing smart contracts with assertions only is not enough. Some test cases require manipulating the
+state of the blockchain, as well as checking for reverts and events. For that reason, Protostar provides a set of
+cheatcodes.
```

Comment:
This does not have much sense in current setting. You are basically unable to test anything contract related without cheatcodes

Reply:
What do you mean exactly? I basically copied the description from the current documentation. I can rewrite it but not sure what should be changed